opentelemetry/src/main/scala/zio/telemetry/opentelemetry/ContextPropagation.scala

```diff
@@ -35,9 +35,7 @@ private[opentelemetry] object ContextPropagation {
     carrier: C,
     getter: HttpTextFormat.Getter[C]
   ): UIO[Span] =
-    ZIO.uninterruptible {
-      UIO(httpTextFormat.extract(Context.ROOT, carrier, getter)).map(TracingContextUtils.getSpan)
-    }
+    UIO(httpTextFormat.extract(Context.ROOT, carrier, getter)).map(TracingContextUtils.getSpan)
```

Comment:
This one must be uninterruptible as it sets and then gets a ThreadLocal. Therefore it absolutely must run on the same thread without anyone touching the ThreadLocal in between.

Reply:
It must have been really late last night :sweat_smile: , as I discover now I misread your comment and accidentally removed this `uninterruptible`. Sorry for the inconvenience :slightly_smiling_face: , should be fixed again